service/arm_templates/security_groups_setup.json

```diff
@@ -193,68 +180,68 @@
           {
             "name": "defaultInClusterRule",
             "properties": {
-              "description": "Default rule that allows any traffic within the workers' subnet.",
+              "description": "Default rule that allows any traffic within the worker subnet.",
               "protocol": "*",
               "sourcePortRange": "*",
               "destinationPortRange": "*",
-              "sourceAddressPrefix": "[parameters('subnetWorkersCIDR')]",
-              "destinationAddressPrefix": "[parameters('subnetWorkersCIDR')]",
+              "sourceAddressPrefix": "[parameters('workerSubnetCidr')]",
+              "destinationAddressPrefix": "[parameters('workerSubnetCidr')]",
               "access": "Allow",
               "direction": "Inbound",
               "priority": "4094"
             }
           },
           {
-            "name": "workersLBToIngressSecurePort",
+            "name": "workerLBToIngressSecurePort",
             "properties": {
-              "description": "Allow the workers' load balancer to reach the ingress controller secure port.",
+              "description": "Allow the worker load balancer to reach the ingress controller secure port.",
               "protocol": "tcp",
               "sourcePortRange": "*",
               "destinationPortRange": "[parameters('kubernetesIngressSecurePort')]",
-              "sourceAddressPrefix": "[parameters('loadBalancerWorkersCIDR')]",
-              "destinationAddressPrefix": "[parameters('subnetWorkersCIDR')]",
+              "sourceAddressPrefix": "[parameters('workerLoadBalancerCidr')]",
+              "destinationAddressPrefix": "[parameters('workerSubnetCidr')]",
               "access": "Allow",
               "direction": "Inbound",
               "priority": "3900"
             }
           },
           {
-            "name": "workersLBToIngressInsecurePort",
+            "name": "workerLBToIngressInsecurePort",
             "properties": {
-              "description": "Allow the workers' load balancer to reach the ingress controller insecure port.",
+              "description": "Allow the worker load balancer to reach the ingress controller insecure port.",
               "protocol": "tcp",
               "sourcePortRange": "*",
               "destinationPortRange": "[parameters('kubernetesIngressInsecurePort')]",
-              "sourceAddressPrefix": "[parameters('loadBalancerWorkersCIDR')]",
-              "destinationAddressPrefix": "[parameters('subnetWorkersCIDR')]",
+              "sourceAddressPrefix": "[parameters('workerLoadBalancerCidr')]",
+              "destinationAddressPrefix": "[parameters('workerSubnetCidr')]",
               "access": "Allow",
               "direction": "Inbound",
               "priority": "3800"
             }
           },
           {
-            "name": "workersLBToKubelet",
+            "name": "workerLBToKubelet",
```

Comment:
I don't think this rule is needed. The kubelet port is only used by k8s and shouldn't be exposed by the LB. The master already has access because traffic is allowed between the master and worker nodes.